lib/store/index.js

```diff
@@ -139,6 +148,12 @@ export class Store {
   updateEditor(editor: ?atom$TextEditor) {
     this.editor = editor;
     this.setGrammar(editor);
+
+    if (this.globalMode && this.kernel && editor) {
+      const fileName = editor.getPath();
+      if (!fileName) return;
+      this.kernelMapping.set(fileName, this.kernel);
+    }
```

Comment:
Files with an associated kernel still need to be tracked in `kernelMapping` for things like Kernel Monitor view. This ensures all editors with kernels associated are tracked.